build.gradle

```diff
@@ -6,9 +6,8 @@ buildscript {
       'kotlin': '2.2.0',
       'autoService': '1.1.1',
 
-      // Update WrongTimberUsageDetectorTest#innerStringFormatWithStaticImport when >= 7.1.0-alpha07
       'androidPlugin': '8.11.0',
-      'androidTools': '30.0.0',
+      'androidTools': '31.10.0',
```

Comment:
You can also update the renovate config to group the AGP updates like https://github.com/JakeWharton/diffuse/blob/4f672e57199515ecc45a753d4a385463b1b5d4d3/.github/renovate.json5#L6-L13

Reply:
I'm not sure what this means, or what it would do. Is this desirable in some way?

Reply:
#552